doc/source/support/frequently-asked-questions.rst

```diff
@@ -134,6 +134,37 @@ Achieving the same result in Markdown often requires raw HTML, which is less rea
 
 Switching to ``README.rst`` at Level 5 helps users appreciate the formatting power of ``.rst`` and serves as a stepping stone toward writing full documentation in ``.rst`` as part of the ``scikit-package`` documentation standard.
 
+.. _faq-set-default-prompt-value:
+
+How can I change the default values that appear in the prompt when creating projects in level 3,4,5?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+For example, to change some default values prompted when creating a new package in level 5, please follow the quick steps below:
+
+1. create ``~/.skpkgrc`` by runing ``touch ~/.skpkgrc``.
+2. copy and paste the following snippets.
+
+.. code-block:: json
+
+  {
+      "default_context":
+          {
+	       "maintainer_name": "Sangjoon Lee",
+               "project_needs_c_code_compiled": ["No", "Yes"],
+	  }
+  }
+3. run ``package create public``.
+
+
+As shown above, you can enter all the level 3, 4 and 5 default values in ``default_context``. For list variables like ``project_needs_c_code_compiled``, you should only swap the order of the elements. The element in the first order will be the default value appeared in the prompt.
```

Comment:
let's not use the phrase "you should only swap"

instead, it's a guide. So, we are here to guide them, not recommend them. 

It's most likely that ALL users will set "No" for the "list variables" so let's not add this "project_needs_c_code_compiled" but only use my example I used in the PR